Options.lua

```diff
@@ -517,6 +518,17 @@ function WarpDeplete:InitOptions()
       }, { order = 4 }),
 
       bars = group("Bars", false, {
+        {
```

Comment:
This option should be in the "Texts" category, since the objectives are shown as text and not as bars. You can also use the utility functions for options, so it should look like this:

```lua
toggle("Show objectives", "showObjectives", "UpdateObjectivesDisplay", { width = "full" })
```